cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -36,62 +36,72 @@
 */
 
 .m-social-media {
-  display: inline-block;
-
-  // This could be wrapped in the `m-social-media__share` modifier,
-  // but it's ok on its own too.
-  &_heading {
-    .heading-5();
-    display: inline-block;
-    // 0.25em subtracted to compensate for inline spacing
-    padding-right: unit(@grid_gutter-width / @font-size - 0.25em, em);
-    margin-bottom: 0;
-    vertical-align: middle;
-  }
+  display: inline-flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  gap: unit(10px / @base-font-size-px, rem);
 
   .m-list {
-    display: inline-block;
-    vertical-align: middle;
-
-    .m-list_item {
-      // 0.25em subtracted to compensate for inline spacing
-      margin-right: unit(
-        (@grid_gutter-width / 2) / @base-font-size-px - 0.25em,
-        em
-      );
-      margin-bottom: 0;
-    }
-
-    .m-list_item:last-child {
-      margin-right: 0;
-    }
+    display: inline-flex;
+    gap: unit(10px / @base-font-size-px, rem);
   }
 
-  &_icon {
+  .m-social-media_icon {
     color: @gray-dark;
-    font-size: unit(@grid_gutter-width / @base-font-size-px, em);
+    font-size: unit(@grid_gutter-width / @base-font-size-px, rem);
     line-height: 1;
     .u-link__colors( @gray-dark, @pacific-80 );
     .u-link__no-border();
   }
 
-  &_print {
-    padding-left: unit((@grid_gutter-width / 2) / @base-font-size-px, em);
-    border-left: 1px solid @black;
-  }
+  //Only for elements in "share" (m-social-media__share) section
+  &__share {
+    .m-social-media_heading {
+      .heading-5();
+      margin: auto 0;
+    }
 
-  // Hide on print.
-  .respond-to-print( {
-    & {
-      display: none;
+    .m-list {
+      .m-social-media_print {
+        padding-left: unit(10px / @base-font-size-px, rem);
+        border-left: 1px solid @black;
+      }
     }
-  } );
+
+    // Hide on print.
+    .respond-to-print( {
+      & {
+        display: none;
+      }
+    } );
+  }
 }
 
 .no-js .m-social-media_print {
   display: none;
 }
 
+//fix issue with padding and vertical rule appearing to the left of print icon
+html[lang='ar'] .m-social-media__share {
+  .m-social-media_print {
+    padding-left: 0;
+    border-left: none;
+    padding-right: unit(10px / @base-font-size-px, rem);
+    border-right: 1px solid @black;
+  }
+
+  .m-list_item:first-child {
+    margin-right: 0;
+  }
+}
+
+// Position share & print text and icons on one line on large screens
+.respond-to-min( @bp-sm-min,  {
+  .m-social-media__share {
+    flex-direction: row;
+  }
+} );
```

Comment:
Let's move this above the RTL block (`html[lang='ar'] …`), so that all LTR CSS is above the RTL CSS.